BigCommerce/Integration/02_Particularities.md

```diff
@@ -0,0 +1,196 @@
+# BigCommerce particularities
+
+There are several particularities regarding the *BigCommerce* data that are to be considered when establishing and using the *BigCommerce* connection. These particularities are described in detail below.
+For detailed information on product data in *BigCommerce*, refer to the *BigCommerce* documentation.   
+
+Note that both the *PIM* module and *BigCommerce* use the term "product". However, the *Omni-Channel* module uses the term "offer". The same object is meant in each case.
+
+
+## Offer handling
+
+You have the following options for exchanging offer data between *Actindo* and *BigCommerce*:
+- If you have the *PIM* module in use, via the *Ommni-Channel* module, you can export the product data needed for your *BigCommerce* store. This includes all merchant-specific additional fields from the *PIM* module. 
+- If you have the *PIM* module not in use, you can export *Omni-Channel* offers directly. 
+- The following is relevant if you started using *BigCommerce* before your started using the *Omni-Channel* module. You can initially import your *BigCommerce* product data to *Omni-Channel*.   
+  Note, if you choose this way, you will not be able to import product changes again from *BigCommerce* to *Actindo*. In this case, you must change the offers on the *Actindo* side and export them to *BigCommerce*.  
+  For this reason, it is recommended to manage offer data at *Actindo*.
+ 
+## Export of offers 
+
+The following particularities are relevant when you export your offers from the *Omni-Channel* module to *BigCommerce*.
+
+### Variant options
+
+*BigCommerce* supports multiple types for the selection of variant options on the UI.  For detailed information, see [Configure BigCommerce connection](./01_ManageBigCommerceConnection.md#configure-bigcommerce-connection).
+
+
+### Custom fields
+
+*BigCommerce* supports simple string values only. For this reason, only the following *Actindo* data types are supported as possible custom field values for attributes:
+
+- Text field
+- String
+- Float 
+- Integer
+- Date time
+
+All data types that are no string values are converted to string. 
+The length of a custom field string is limited to 255 characters. 
+
+**To do**:   
+Check your custom fields if they meet the requirements. 
```

Comment:
Gemeint: Check if your custom fields meet the requirements?

Reply:
das Wichtigste zuerst